.github/workflows/ipa.yml

```
@@ -87,25 +89,69 @@ jobs:
+
+          unzip "$DECRYPTED_IPA" -d "$TMP_DIR"
+          APP_PATH=$(find "$TMP_DIR/Payload" -type d -name "*.app" | head -n 1)
+          PLIST="$APP_PATH/Info.plist"
+          EXECUTABLE_NAME=$(/usr/libexec/PlistBuddy -c "Print :CFBundleExecutable" "$PLIST")
+          BIN_PATH="$APP_PATH/$EXECUTABLE_NAME"
+
+          mkdir -p "$APP_PATH/Frameworks"
+          
+          DYLIB_PATH=$(find deb-extract -name "*.dylib" | head -n 1)
+          cp "$DYLIB_PATH" "$APP_PATH/Frameworks/$DYLIB_NAME"
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)